src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -83,7 +83,7 @@
 # since standalone samples can perform more sophisticated response handling
 @private pagedIterableMethodSampleCode(apiMethod, sample)
   {@initCode(sample.initCode)}
-  for ({@apiMethod.listMethod.resourceTypeName} element : {@sampleSyncMethodCall(apiMethod)}.{@apiMethod.listMethod.iterateMethodName}()) {
+  for ({@apiMethod.listMethod.resourceTypeName} responseItem : {@sampleSyncMethodCall(apiMethod)}.{@apiMethod.listMethod.iterateMethodName}()) {
```

Comment:
Hmm, if we already have `getSampleResponseVarName`, then maybe we shouldn't hard-code these variable names here. WDYT, @pongad ?

Let's fix this in a separate PR. For now, add a `#FIXME` near the top of this file.

Reply:
IIUC `getSampleResponseVarName` is only a utility function, and we don't actually save this somewhere in the model or view-model. And therefore hard-coding in snippets is the only way with current infrastructure. But I agree with you that we should fix this.